src/renderer/util/fileSystem.js

```diff
@@ -173,7 +194,15 @@ export const uploadImage = async (pathname, image, preferences) => {
       if (size > MAX_SIZE) {
         notification()
       } else {
+<<<<<<< HEAD
```

Comment:
Merge conflict. There are a few issues in the code, please run `yarn lint` again and fix these.